### DISCUSSION TITLE
Migrating from v8 multiple projects

### DISCUSSION THREAD

**Discussion by NigmaX** [directus/directus#13382] [Archive]
we are currently using Directus v8, with multiple projects configuration. (we have a dropdown on the login screen to select the wanted project)
we want to upgrade to v9, but I couldn't find it in the docs, and migration guides do not show any support for multiple projects on the v9 version.

so my questions are:
1) does Directus v9 supports multiple projects at all?
2) if not, what would be the best approach for migrating multiple projects into v9.

I really appreciate any help you can provide.

**Comment by rijkvanzanten** [comment]
> does Directus v9 supports multiple projects at all?
> if not, what would be the best approach for migrating multiple projects into v9.

No, every instance of Directus connects to one database. v9 instances are wayyy lighter to run, so the recommended approach is to run multiple copies of Directus (one per project) 👍🏻

**Reply by 3zzy** [reply]
@rijkvanzanten I wonder if it's going to change with "Multiple Datasources" feature in the roadmap for Q4 2022 which "more formally begin to support multiple datasources.". 

I mean would it still essentially be the same approach you described - separate instance per db - or different? Can I read more about this somewhere because I'm looking to create a multi-site network.

Thanks

**Reply by rijkvanzanten** [reply]
There's currently no plan to support multiple scoped projects in the same installation as you've seen in v8. The multiple datasources feature would allow you to hookup multiple databases to the same installation, but it won't scope the app or api per data source
